src/pages/RFOX/hooks/useStakingBalanceOfQuery.ts

```diff
@@ -68,5 +68,5 @@ export const useStakingBalanceOfQuery = <SelectData = StakingBalanceOf>({
     select,
   })
 
-  return stakingBalanceOfQuery
+  return { ...stakingBalanceOfQuery, queryKey }
```

Comment:
Not using `wagmi/query`'s `useQuery` variant anymore, so we expose queryKey the same like `wagmi/query` does